ethereum/spec/src/test/java/tech/pegasys/teku/spec/datastructures/state/beaconstate/versions/capella/BeaconStateCapellaTest.java

```diff
@@ -32,6 +33,13 @@ protected BeaconStateSchema<BeaconStateCapella, MutableBeaconStateCapella> getSc
     return BeaconStateSchemaCapella.create(specConfig);
   }
 
+  @Test
```

Comment:
Either forgot to delete or we need some tests here :)

Reply:
forgot to delete my hack